.github/workflows/compose_run.yaml

```
@@ -73,6 +77,14 @@ jobs:
         ci_env=`bash <(curl -s https://codecov.io/env)`
         docker-compose -f docker-compose.test.yml exec -T -e CI=true $ci_env ${{ inputs.application-name }} ${{ inputs.cmd}}
 
+    - name: upload artifact
+      # uploads only if run failed
+      if: ${{ failure() && inputs.artifacts_path !=  '' }}
```

The default for `artifacts_path` is declared as `null`. Is this condition applicable to that case, or would this be safer? :
```suggestion
      if: ${{ failure() && inputs.artifacts_path }}
```